package.json

```diff
@@ -3,10 +3,11 @@
     "@primer/css": "^20.8.0",
     "ace-builds": "^1.33",
     "katex": "^0.16.10",
-    "mermaid": "^10.9",
+    "mermaid": "^11.1.0",
     "mousetrap": "^1.6.5"
   },
   "engines": {
     "node": ">=16.15.0"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
@dometto Any idea why this `packageManager` key is being added?

Reply:
It looks like the author is using [Corepack](https://github.com/nodejs/corepack) as a JavaScript package manager and this is being added automatically by the tool. I did a web search for the SHA512 hash and see numerous webpages referencing it.

Reply:
Yep! I was following the yarn setup guide on my machine. 

(Haven't done any JS since about 2016, so was a bit surprised how the ecosystem has changed).